modules/extensions-support/src/main/java/org/mule/module/extension/internal/util/IntrospectionUtils.java

```diff
@@ -381,4 +386,20 @@ public static String getSourceName(Class<? extends Source> sourceType)
 
         return sourceType.getSimpleName();
     }
+
+    public static java.util.Optional<ParameterModel> getContent(OperationModel operation)
```

Comment:
getContent -> getContentParameter

Does it make sense to move Metadata related introspections to its own class? Main reason for that would be to keep INtrospectionUtils from overgrowing... consider it, not a mandate. If only a few methods can be extracted then probably not worth it for now